cmd/fuseml_core-cli/main.go

```diff
@@ -99,8 +99,8 @@ func usage() {
 Usage:
     %s [-host HOST][-url URL][-timeout SECONDS][-verbose|-v] SERVICE ENDPOINT [flags]
 
-    -host HOST:  server host (localhost). valid values: localhost
-    -url URL:    specify service URL overriding host URL (http://localhost:8080)
+    -host HOST:  server host (0.0.0.0). valid values: 0.0.0.0
```

Comment:
nit/Side note: I would use the code parsing the CLI arguments to generate usage string, instead of having some text that can be easily outdated. cobra isn't a bad library, though a bit big.

Reply:
I also had cobra in mind, have seen it used in several projects and look easy to use and feature rich. this code is still based on the original generated by goa, but we'll have to improve that at some point.

Reply:
agreed, at some point we need to improve the CLI. This is just a starting point.